include/eve/memory/pointer.hpp

```diff
@@ -22,14 +22,12 @@ namespace eve::detail
 
   template <typename U, typename T, std::size_t size>
   EVE_FORCEINLINE auto ptr_cast(eve::aligned_ptr<T, size> p)
-    requires (sizeof(U) == sizeof(T))
```

Comment:
shouldn't it just be std::convertibel_to<U*,T*> ?

Reply:
Nah. It's a C style cast for aligned ptr

Reply:
OK